### ISSUE TITLE
7.2 - Gilnean voices OGG (117)

### ISSUE THREAD

**Issue by hoxton22** [bloerwald/LegionFiles#31]
Heeeem, that's strange but it seems that finally i can find more... only after doing an issue with only 6 path... i guess... well anyway.
Sorry for sending 2 issues, but when i've posted the first, i didn't expected finding more paths today !

https://ghostbin.com/paste/g6r96

**Comment by Marlamin** [comment]
Wow, 117 new filenames! Very nice.

**Comment by hoxton22** [comment]
Thanks ! appreciated it ! Also thanks to you and tom_rus for your contributions :)

**Comment by justMaku** [comment]
Thanks for your research 👍

**Comment by Marlamin** [comment]
FYI @hoxton22 seems like there are 9 missing files still which also seem to be gilnean voices. 

(See red entries on page [29330](https://bnet.marlam.in/files.php#search=&page=29330), [29332](https://bnet.marlam.in/files.php#search=&page=29332) and [29332](https://bnet.marlam.in/files.php#search=&page=29333))

Maybe you'll be able to find those too!

**Comment by hoxton22** [comment]
Allright, i will check this out

**Comment by hoxton22** [comment]
I got it !
It seems i didn't have their filedataid <=> soundId <=> "SoundName" while reading in SoundEntry and SoundKitEntry

![image](https://cloud.githubusercontent.com/assets/13040767/23625322/0df0d510-02a9-11e7-845a-39dff018d697.png)


sound/character/pcgilneanfemale/vo_gilneanfemale_main_casterwindup_01.ogg
sound/character/pcgilneanfemale/vo_gilneanfemale_main_casterwindup_02.ogg
sound/character/pcgilneanfemale/vo_gilneanfemale_main_casterwindup_03.ogg
sound/character/pcgilneanfemale/vo_gilneanfemale_main_casterwindup_04.ogg
sound/character/pcgilneanfemale/vo_gilneanfemale_main_casterwindup_05.ogg
sound/character/pcgilneanfemale/vo_gilneanfemale_main_casterwindup_06.ogg
sound/character/pcgilneanmale/vo_gilneanmale_main_casterwindup_01.ogg
sound/character/pcgilneanmale/vo_gilneanmale_main_casterwindup_02.ogg
sound/character/pcgilneanmale/vo_gilneanmale_main_casterwindup_03.ogg
sound/character/pcgilneanmale/vo_gilneanmale_main_casterwindup_04.ogg
sound/character/pcgilneanmale/vo_gilneanmale_main_casterwindup_05.ogg

**Comment by Marlamin** [comment]
Awesome! Added them to new pull request.
